### ISSUE TITLE
Add an example of Quarkus using jib for containerizing an application

### ISSUE THREAD

**Issue by yrashish** [GoogleContainerTools/jib#3017]
Let me know if it makes sense to add Quarkus example to the existing examples.

**Comment by chanseokoh** [comment]
Absolutely. We intend to have good support for Quarkus. I'm curious what type of example it would be or it could be a representative example or promoted as a recommended way. We have a Jib Quarkus extension, but I can imagine you can build an image in a different way too.

**Comment by yrashish** [comment]
@chanseokoh I was thinking of using the Jib Qurakus [extension](https://github.com/GoogleContainerTools/jib-extensions/tree/master/first-party/jib-quarkus-extension-maven) for the example. But it seems Qurakus has its own extension [framework](https://quarkus.io/guides/container-image#jib) as well. Let me know which makes the most sense as an example or I can try to cover both separately?

**Comment by chanseokoh** [comment]
In this repo, we should have an example that builds an image through the Jib build plugins on a Quarkus project. That is, using the Jib Quarkus extension. The way Quarkus using the Jib library is out of our control, but maybe we can also briefly document Quarkus' built-in Jib feature. In any case, our Quarkus support through the Jib build plugins are certainly different and more flexible, and we'd like to promote our way.

**Comment by yrashish** [comment]
@chanseokoh Makes sense to me. I will raise a PR once done.

**Comment by ldetmer** [comment]
@yrashish I see you still have an open PR for this.  Please let me know if you plan on addressing comments, otherwise we can close this issue

**Comment by diegomarquezp** [comment]
Closing due to lack of activity. Feel free to reopen as needed.
